cli/cmd/uninstall.go

```diff
@@ -10,13 +10,12 @@ import (
 	"time"
 
 	"github.com/odigos-io/odigos/api/k8sconsts"
-	"github.com/odigos-io/odigos/common/envOverwrite"
```

Comment:
this is the motivation, to stop using "github.com/odigos-io/odigos/common/envOverwrite" everywhere and git rid of it